docs/data/sdks/ampli-overview.md

```diff
@@ -0,0 +1,234 @@
+---
```

Comment:
As part of this PR I am considering moving all Ampli docs from `Data -> Sources -> Ampli` on https://www.docs.developers.amplitude.com/data/ampli/  to `Ampli` on https://www.docs.developers.amplitude.com/data/sdks/ampli 

WDYT?

Reply:
I was about to ask you. And love this move! Currently Ampli is buried so deep and disconnected from SDKs

Reply:
It's it possible to have a shared directory and make it show on both Data and SDK?

Reply:
@yuhao900914 I was planning on making it link from Data -> Sources -> Ampli similar to Data -> Sources -> SDK. Not sure how else to make it show in both places without a lot of complication.

Reply:
I saw we show SDKS under `www.docs.developers.amplitude.com/documentation-home/  -> Analytics -> SDKs` and also under https://www.docs.developers.amplitude.com/documentation-home/ -> Data -> Source -> SDKs. Maybe we can do the same thing for Ampli.

Reply:
Please don't move docs around in the directories unless you have to, as we don't want to break existing links. If you do have to, we should get in the habit of setting up redirects.

It's a lot easier just to move the file in the nav than move the file around in the directory structure, which changes the end link and will cause 404s for anyone with an existing link to a document.

Reply:
For now I kept `Data -> Sources -> Ampli` and also added `SDKs -> Ampli` by sharing the page content using an include. Will leave major restructure for another PR.